src/main/java/net/dries007/tfc/common/blockentities/rotation/WindmillBlockEntity.java

```diff
@@ -220,4 +227,31 @@ public Node getRotationNode()
     {
         return node;
     }
+
+    public boolean hasFullIdenticalSet()
+    {
+        ItemStack stack = inventory.getStackInSlot(0);
+
+        // catch anything that's not a windmill blade
+        for (int i = 1; i < SLOTS; i++)
+        {
+            if (!inventory.getStackInSlot(i).is(TFCTags.Items.ALL_WINDMILL_BLADES))
+            {
+                return false;
+            }
+        }
```

Comment:
This is unnecessary because

1. It should be prevented from the `isItemValid` implementation above, so this should never catch anything, and
2. If it _does_, it's not actually preventing cast errors later in the method - the tag and then subsequent cast are not guaranteed to be 1-1 in the presence of a datapack.